_data/social.yml

```diff
@@ -158,7 +158,12 @@ websites:
       url: https://pinterest.com
       twitter: Pinterest
       img: pinterest.png
-      tfa: No
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      exceptions:
+          text: "Software tokens can only be provided through Authy at this time."
```

Comment:
It doesn't sound like they are planning to support other totp clients. it seems like both sms and tokens are generated completely within twillo. Unless i'm misreading their announcement?